pkg/commands/catalog/mirror/mirror.go

```diff
@@ -97,13 +107,62 @@ func Mirror(options Options) error {
 	}
 	images = imageUtil.AddDefaultRegistries(images, options.DefaultRegistry)
 	images = removeDuplicates(images)
```

Comment:
err does not appear to get set for the call to removeDuplicates

Reply:
Thanks! I removed the unecessary error check.